stl/inc/experimental/generator

```diff
@@ -33,18 +32,16 @@ _STL_DISABLE_CLANG_WARNINGS
 _STD_BEGIN
 
 namespace experimental {
-
-    template <typename _Ty, typename _Alloc = allocator<char>>
+    // NOTE WELL: _CPPUNWIND currently affects the ABI of generator.

```

Comment:
Should we `#pragma detect_mismatch` in this header? (OK if the answer is "no" or "we can file an enhancement issue".)

Reply:
It feels like overkill to guard against mixing kernel and non-kernel TUs with detect_mismatch. It made a lot of sense to have a comment here when I was expecting to support `/EHs-` with the `/kernel` layout - linking `/EHs` and `/EHs-` TUs seems more likely than `/kernel` and non-`/kernel` - but now I'm not sure the comment adds value at all.